pkg/reconciler/pipelinerun/resources/pipelineref_test.go

```diff
@@ -764,15 +778,15 @@ func TestGetPipelineFunc_VerifyError(t *testing.T) {
 			}
 			fn := resources.GetPipelineFunc(ctx, k8sclient, tektonclient, tc.requester, pr, vps)
 
-			resolvedPipeline, resolvedRefSource, err := fn(ctx, pipelineRef.Name)
-			if !errors.Is(err, tc.expectedErr) {
-				t.Errorf("GetPipelineFunc got %v, want %v", err, tc.expectedErr)
+			_, _, vr, err := fn(ctx, pipelineRef.Name)
+			if err != nil {
+				t.Errorf("want err nil but got %v", err)
 			}
-			if d := cmp.Diff(resolvedPipeline, (*v1beta1.Pipeline)(nil)); d != "" {
-				t.Errorf("resolvedPipeline did not match: %s", diff.PrintWantGot(d))
+			if !errors.Is(vr.Err, tc.expectedErr) {
+				t.Errorf("GetPipelineFunc got %v, want %v", err, tc.expectedErr)
 			}
-			if resolvedRefSource != nil {
-				t.Errorf("got %v, but expected refSource is nil", resolvedRefSource)
+			if vr.VerificationResultType != trustedresources.VerificationError {
+				t.Errorf("VerificationResultType mismatch, want %d got %d", trustedresources.VerificationError, vr.VerificationResultType)
 			}
```

Comment:
This part can be cleaned up by writing the custom Comparer option as mentioned above.